crates/rust-analyzer/src/cli/analysis_stats.rs

```diff
@@ -193,22 +200,82 @@ impl flags::AnalysisStats {
         Ok(())
     }
 
-    fn lower_mir(&self, db: &RootDatabase, funcs: &[Function]) {
-        let all = funcs.len();
+    fn run_data_layout(&self, db: &RootDatabase, adts: &[hir::Adt], verbosity: Verbosity) {
```

Comment:
Do we want a progress bar here?

Reply:
I think it wouldn't help much, as it is pretty fast currently (I didn't measure it, but my feeling is that it's less than 10 seconds). But adding a timer maybe helps?